server/storage/mysql_test.go

```diff
@@ -22,15 +22,15 @@ func init() {
 		logrus.Fatal("MYSQL environment variable not set")
 	}
 
-	for i := 0; i < 30; i++ {
+	for i := 0; i <= 30; i++ {
 		gormDB, err := gorm.Open("mysql", dburl)
 		if err == nil {
 			err := gormDB.DB().Ping()
 			if err == nil {
 				break
 			}
 		}
-		if i == 29 {
+		if i == 30 {
```

Comment:
This was already correctly retrying 30 times. The updated version will attempt 31 times. The iteration is zero indexed so when you reach line 33, `gorm.Open` at line 26 will have been called `i + 1` times for any given iteration. Therefore, when `i == 29`, `gorm.Open` has been attempted 30 times.